pkgs/top-level/lua-packages.nix

```diff
@@ -132,6 +138,43 @@ with self; {
     };
   };
 
+  awesome-wm-widgets = toLuaModule(stdenv.mkDerivation rec {
+    pname = "awesome-wm-widgets";
+    version = "a83ba6ff7";
```

Comment:
Please never use short shas anywhere longterm. Either use the full sha or a tag. Also version should be in unstable-XXXX-XX-XX format.

Reply:
ah ok, I was wondering about what to use for a version. Should be done now.